tests/fixtures/TagLib.Tests.FileFormats/OpusFormatTest.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using NUnit.Framework;
+using TagLib;
+
+namespace TagLib.Tests.FileFormats
+{
+    [TestFixture]
+    public class OpusFormatTest : IFormatTest
+    {
+        private static string sample_file = "samples/sample.opus";
```

Comment:
Sorry about that. I should have known better. That's what I get for doing this at 2 am :(